changes/ocd-3563.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS openchpl.listing_validation_report;
+
+CREATE TABLE openchpl.listing_validation_report (
+  id                        bigserial NOT NULL,
+  chpl_product_number       text NOT NULL,
+  product_name              text NOT NULL,
```

Comment:
Same question for product_name and certification_status_name, I guess. I get that it's faster to store it than the join the information from the details view but I'm just wondering about it being wrong as listings are updated.